sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md

```diff
@@ -132,15 +132,15 @@ After you have linked Configuration Manager to OMS, you can add or remove collec
       &lt;value>https://management.usgovcloudapi.net/&lt;/value>  
       &lt;/setting>
 
-  Change the value for the setting name *FairFaxAuthorityResource* to be equal to "https://login.microsoftonline.com/"
+  Change the value for the setting name *FairFaxAuthorityResource* to be equal to "https://login.microsoftonline.us/"
 
   - **Original:**
     &lt;setting name="FairFaxAuthorityResource" serializeAs="String">   
     &lt;value>&lt;/value>
 
 	- **Edited:**
     &lt;setting name="FairFaxAuthorityResource" serializeAs="String">   
-    &lt;value>https://login.microsoftonline.com/&lt;/value>
+    &lt;value>https://login.microsoftonline.us/&lt;/value>

```

Comment:
Verified. I can accept with the other revision. Thanks!

Reply:
@mattbriggs want me to merge?

Reply:
@arob98 sure. Thanks! Matt